### PR TITLE
Feature/131/mid section refactor

### PR DESCRIPTION
**Pull request by Aleksandandar-Nedelkovski**
### Impacted areas in the application: mid-section

List general components of the application that this PR will affect: 
* 

PR checklist
- [x] I included  a screenshot for FE changes
- [x] I have linked the PR to a Zenhub ticket
- [x] I have checked for merge conflicts

Laptop
laptop

Mobile
mobile

Tablet
tablet

### CODE REVIEW

File: frontend/src/components/home/midsection/lower_mid.jsx [hunk 1]
```diff
@@ -1,110 +1,121 @@
...
-              </li>
-            </ul>
-          </Row>
-          <Row className="justify-content-md-center">
+              </Col>
+            </Row>
+          </Col>
+          <Col xs={1} sm={1} md={1} lg={2} xl={2}></Col>
+        </Row>
+        <Row className="justify-content-md-center">
```
Comment (sudo-eddy) [inline frontend/src/components/home/midsection/lower_mid.jsx]: There's a bit of overlapping with these images on screen size 1020x280

File: frontend/src/components/home/midsection/upper_mid.jsx [hunk 2]
```diff
@@ -1,36 +1,36 @@
...
     return (
-      <div>
-        <div className="container">
-          <h2 className="text-center heading">Who Are We?</h2>
-
-          <div>
-            <p className="lineParagraph">
+      <Container fluid={true}>
+        <Row className="justify-content-md-center">
+          <Col xs={1} sm={1} md={2} lg={2} xl={2}></Col>
```
Comment (sudo-eddy) [inline frontend/src/components/home/midsection/upper_mid.jsx]:
```suggestion
          
```

File: frontend/src/components/home/midsection/mid.css [hunk 3]
```diff
@@ -1,18 +1,7 @@
-.bottom-text {
```
Comment (sudo-eddy) [inline frontend/src/components/home/midsection/mid.css]: it looks like you have declared the same font-family several times, I would probably use the `*` css selector at the top of the file and just declare the `Medium` type of font for the elements you want as Medium

File: frontend/src/components/home/midsection/upper_mid.jsx [hunk 4]
```diff
@@ -1,36 +1,36 @@
...
+            </div>
+            <div>
+              <p className="bottom-text upper-mid-sectiontext">
+                Young Masterbuilders in Motion, Inc., is a 501(c)(3)nonprofit
+                organization and your contributions are deductible to the full
+                extent of the law.
+              </p>
+            </div>
+          </Col>
+          <Col xs={1} sm={1} md={2} lg={2} xl={2}></Col>
```
Comment (sudo-eddy) [inline frontend/src/components/home/midsection/upper_mid.jsx]:
```suggestion
          
```

File: frontend/src/components/home/midsection/lower_mid.jsx [hunk 5]
```diff
@@ -1,110 +1,121 @@
...
-          <div>
-            <p className="lineParagraph">
+      <Container fluid={true}>
+        <Row className="lower-mid">
+          <Col xs={1} sm={1} md={2} lg={2} xl={2}></Col>
+          <Col xs={10} md={8} md={8} lg={8} xl={8}>
+            <h2 className="text-center heading">What's New?</h2>
+            <b>
+              <h2 className="text-center sub-heading">
+                Mar 9 2019 - Let's Do 300! Emergency Care Package Drive
```
[the remark this answers is not in the document]
Comment (Aleksandandar-Nedelkovski) [reply frontend/src/components/home/midsection/lower_mid.jsx]: @chitownhope said to keep current state.

**Pull request merged**